setup/sql/ilias3.sql

```diff
@@ -5262,7 +5262,7 @@ INSERT INTO `ctrl_classfile` VALUES ('ilobjtaxonomygui','Services/Taxonomy/class
 INSERT INTO `ctrl_classfile` VALUES ('ilobjtaxonomyadministrationgui','Services/Taxonomy/classes/class.ilObjTaxonomyAdministrationGUI.php','','','3x');
 INSERT INTO `ctrl_classfile` VALUES ('iltaxonomyexplorergui','Services/Taxonomy/classes/class.ilTaxonomyExplorerGUI.php','','','3y');
 INSERT INTO `ctrl_classfile` VALUES ('ilhistorytablegui','Services/History/classes/class.ilHistoryTableGUI.php','','','3z');
-INSERT INTO `ctrl_classfile` VALUES ('ilobjfileaccesssettingsgui','Services/WebDAV/classes/class.ilObjFileAccessSettingsGUI.php','','','40');
+INSERT INTO `ctrl_classfile` VALUES ('ilobjfileaccesssettingsgui','Modules/File/classes/class.ilObjFileAccessSettingsGUI.php','','','40');
```

Comment:
@rsheer this is not needed and will be done when a new release is published. you can revert the file.